distmv/normal.go

```diff
@@ -267,6 +286,15 @@ func (n *Normal) Rand(x []float64) []float64 {
 	return x
 }
 
+// SetMean changes the mean of the normal distribution. SetMean panics if the
+// length of mu does not equal the dimension of the normal distribution.
```

Comment:
You use `len(·)` in the other comments, I think it is slightly easier to read but up to you.